text/0000-marketing-releases.md

```diff
@@ -0,0 +1,58 @@
+- Start Date: 2017-10-09
+- RFC PR: (leave this empty)
+- Ember Issue: (leave this empty)
+
+# Summary
+
+Add "named releases" to allow for easier marketing efforts while simultaneously maintaining Ember's strong semver guarantees.
+
+# Motivation
+
+Ember's commitment to semver is valuable. The practice of introducing new features in minor versions and major versions only removing deprecated functionality makes for a straightforward and predictable upgrade path.
+
+Unfortunately, this is not the norm throughout the JavaScript community, despite the wide usage of semver version numbering via tooling like npm. This leads to a marketing problem, as evidenced by some of the reaction to the Ember 3.0 announcement.
+
+For most other projects, a semver major bump is a big marketing event. Culturally, the expectations are to see great new features. For Ember, semver major bumps are comparatively lackluster - simply removing functionality that you probably have stopped using anyway.
+
+This may lead developers who may not fully understand our semver practices to assume that the apparently lackluster 3.0 announcement is indicative of a lackluster project (when in reality, it's the opposite).
+
+This RFC proposes that we attempt to decouple semver version numbers from our marketing efforts, and adopt a different set of "marketing releases" which can be used to generate interest around new features. The goal is to find a solution that allows us to keep our strong semver guarantees while also allowing for effective marketing of Ember's new features and advancements.
+
+# Detailed design
+
+> Note: this is one possible approach to marketing releases - others are discussed in the Alternatives section below
```

Comment:
I think this is explicit in the RFC design, so maybe unnecessary?

Reply:
I was trying to indicate that there are really two sides to this RFC - "marketing releases" and "named releases". Named releases are one possible implementation of marketing releases, but you can be 👍  on marketing releases and 👎  on named releases. But if that's already clear / implied by the RFC process, happy to pull this.